core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -551,12 +551,16 @@ class TestSuiteWriter {
 
 
         lines.block {
+
             if (format.isJavaScript()) {
+                //TODO add resetDatabase
                 addStatement("await $controller.resetStateOfSUT()", lines)
             } else if (format.isJavaOrKotlin()) {
+                addStatement("$controller.resetDatabase()", lines)
```

Comment:
so, are we always adding `resetDatabase()`? shouldn't that depend on whether we handle or not a smart reset? (ie, if not, it should not be there). or is this handled inside `resetDatabase()`? clarify

Reply:
but, even in this latter case, it would be confusing to see a call that does nothing in the generated tests